apps/cloudbroker/base/cloudapi__users/methodclass/cloudapi_users.py

```diff
@@ -89,6 +89,7 @@ def setData(self, data, **kwargs):
                 userdata = json.loads(user.data)
             except:
                 userdata = {}
+            data = json.loads(data)
```

Comment:
this shouldnt be needed if a proper post is done this should already have been deserialized